libs/barista-components/stacked-series-chart/src/stacked-series-chart-path.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { AfterContentInit, Directive, Input, TemplateRef } from '@angular/core';
+import { DtOverlayTrigger } from '@dynatrace/barista-components/overlay';
+import { DtStackedSeriesChartTooltipData } from './stacked-series-chart.util';
+
+export interface DtStackedSeriesChartOverlayData {
+  name: string;
+  value: number;
+  totalValue: number;
+}
+
+@Directive({
+  selector: '[dt-stacked-series-chart-path]',
+  host: {
+    '(mouseenter)': 'overlayTemplate && _handleMouseEnter($event)',
+    '(mouseleave)': 'overlayTemplate && _handleMouseLeave($event)',
+  },
+})
+export class DtStackedSeriesChartPath
+  extends DtOverlayTrigger<{ $implicit: DtStackedSeriesChartOverlayData }>
```

Comment:
This is a no go... Please do not extend from the overlay trigger. You can use the overlay in the template, but please do not extend from the trigger.